### PR TITLE
CB-11204: Updating windows platform requirement instructions for inst…

### PR DESCRIPTION
**Pull request by rakatyal**
…alling required SDKs

### CODE REVIEW

**Comment by rakatyal** [comment]
@riknoll , @omefire : Please review.

File: www/docs/en/dev/guide/platforms/win8/index.md [hunk 1]
```diff
@@ -80,11 +80,10 @@ Install any edition of
 [Visual Studio](http://www.visualstudio.com/downloads) matching the version
 requirements listed above.
 
-<br/><p align="center"><img src="{{ site.baseurl }}/static/img/guide/platforms/win8/win8_installSDK.png" /></p><br/>
+You would also need to install the tools and SDKs required for the target windows platforms(UWP, 8.1 etc.) from the installer.
```
Comment (riknoll) [inline www/docs/en/dev/guide/platforms/win8/index.md]: I would remove the "you" references. Something like:
> The tools and SDKs for the target Windows platforms (UWP, 8.1, etc.) must also be selected in the installer. They can be found under the "Windows and Web Development" heading

Comment (rakatyal) [inline www/docs/en/dev/guide/platforms/win8/index.md]: Updated.

**Comment by riknoll** [comment]
LGTM! Only other comment is that it looks like you mention UWP but it doesn't show up in the screenshot

**Pull request closed without merging**